tests/run.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -euxo pipefail
+cd "$(dirname "$0")"
+
+rm -rf santas_warehouse
+mkdir -p santas_warehouse
+
+(
+  cd santas_warehouse
+
+  function trek {
+    go run ../.. "${@}"
+  }
+
+  TREK_VERSION=latest \
+  TREK_MODEL_NAME=santas_warehouse \
+  TREK_DATABASE_NAME=north_pole \
+  TREK_DATABASE_USERS=santa,worker \
+  trek init
+
+  trek check
+
+  for file in ../stages/*; do
+    cp "$file" santas_warehouse.dbm
+
+    trek generate "$(basename "$file" | cut -d "-" -f 2 | cut -d "." -f 1)"
+
+    trek check
```

Comment:
Should we compare the generated migration to a fixture/"golden" that we store in the repo?